core/src/main/scala/kafka/log/UnifiedLog.scala

```diff
@@ -683,6 +702,7 @@ class UnifiedLog(@volatile var logStartOffset: Long,
       validateAndAssignOffsets = false,
       leaderEpoch = -1,
       requestLocal = None,
+      verificationState = Optional.empty(),
```

Comment:
We currently don't. 
I guess there is an argument that we could have a marker come in before a fetch response. 

I'm not sure how this would be implemented.

Reply:
I suppose I should confirm we won't have an issue when the verification state is blank -- I will make sure of this.